http4s-blaze-client/src/main/scala/com/avast/server/toolkit/http4s/Http4sBlazeClient.scala

```diff
@@ -0,0 +1,52 @@
+package com.avast.server.toolkit.http4s
+
+import cats.effect.{ConcurrentEffect, Resource, Sync}
+import cats.implicits._
+import cats.{Applicative, Traverse}
+import com.avast.server.toolkit.ssl.{SslContextConfig, SslContextModule}
+import javax.net.ssl.SSLContext
+import org.http4s.client.Client
+import org.http4s.client.blaze.BlazeClientBuilder
+
+import scala.concurrent.ExecutionContext
+import scala.language.higherKinds
+
+object Http4sBlazeClient {
+
+  /** Makes [[org.http4s.client.Client]] (Blaze) initialized with the given config. */
+  def make[F[_]: ConcurrentEffect](config: Http4sBlazeClientConfig, executionContext: ExecutionContext): Resource[F, Client[F]] = {
+    makeClient(config, executionContext)
+  }
+
+  private def makeClient[F[_]: ConcurrentEffect: Applicative](config: Http4sBlazeClientConfig,
```

Comment:
just rename this to `make`

also, no need for `Applicative`, because `ConcurrentEffect` already is